### PR TITLE
[Docs][Connector-V2][HDFS]Refactor connector-v2 docs using unified format HDFS.

### PR DESCRIPTION
**Pull request by lightzhao**
## Purpose of this pull request
subtask of #4572

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by zhilinli123** [comment]
Refer to this structure for changes. Currently there is some standardization of naming conventions and formats. Thank you:
Source:https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/source/Mysql.md
Sink:https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/sink/Mysql.md
image

File: docs/en/connector-v2/sink/HdfsFile.md [hunk 1]
```diff
@@ -1,18 +1,12 @@
...
-
-If you use SeaTunnel Engine, It automatically integrated the hadoop jar when you download and install SeaTunnel Engine. You can check the jar package under ${SEATUNNEL_HOME}/lib to confirm this.
+## Support Those Engines
 
-:::
+> Spark<br/>
+> Flink<br/>
+> Seatunnel Zeta<br/>
 
 ## Key features
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/HdfsFile.md:11]: Key features

File: docs/en/connector-v2/sink/HdfsFile.md [hunk 2]
```diff
@@ -30,183 +24,102 @@ By default, we use 2PC commit to ensure `exactly-once`
...
-Sink plugin common parameters, please refer to [Sink Common Options](common-options.md) for details
+## Description
 
-### max_rows_in_memory [int]
+Output data to hdfs file
 
-When File Format is Excel,The maximum number of data items that can be cached in the memory.
+## Sink Options
+
+|               name               |  type   | required |               default value                |                                                                                                                                                                                                                                       Description                                                                                                                                                                                                                                        |
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/HdfsFile.md:33]: **example**:Name Type Required Default Value

File: docs/en/connector-v2/source/HdfsFile.md [hunk 3]
```diff
@@ -33,233 +27,51 @@ Read all the data in a split in a pollNext call. What splits are read will be sa
...
+## Description
 
-tyrantlucifer#26#male
+Read data from hdfs file system.
 
-```
+## Source Options
 
-If you do not assign data schema connector will treat the upstream data as the following:
+|           name            |  type   | required |    default value    |                                                                                                                                                                  Description                                                                                                                                                                  |
```
Comment (zhilinli123) [inline docs/en/connector-v2/source/HdfsFile.md:36]: Same as above

File: docs/en/connector-v2/sink/HdfsFile.md (hunk 1 above)
Comment (zhilinli123) [inline docs/en/connector-v2/sink/HdfsFile.md:11]: Key Features

**Comment by lightzhao** [comment]
> Refer to this structure for changes. Currently there is some standardization of naming conventions and formats. Thank you: Source:https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/source/Mysql.md Sink:https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/sink/Mysql.md image

update.

File: docs/en/connector-v2/sink/HdfsFile.md [hunk 4]
```diff
@@ -1,18 +1,12 @@
...
-
-If you use spark/flink, In order to use this connector, You must ensure your spark/flink cluster already integrated hadoop. The tested hadoop version is 2.x.
-
-If you use SeaTunnel Engine, It automatically integrated the hadoop jar when you download and install SeaTunnel Engine. You can check the jar package under ${SEATUNNEL_HOME}/lib to confirm this.
+## Support Those Engines
 
-:::
+> Spark<br/>
+> Flink<br/>
+> Seatunnel Zeta<br/>
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/HdfsFile.md]: `SeaTunnel Zeta`

File: docs/en/connector-v2/sink/HdfsFile.md [hunk 5]
```diff
@@ -30,183 +24,128 @@ By default, we use 2PC commit to ensure `exactly-once`
...
+| TINYBLOB<br/>MEDIUMBLOB<br/>BLOB<br/>LONGBLOB<br/>BINARY<br/>VARBINAR<br/>BIT(n)                                                  | BYTES                                                                                                                                               |
+| GEOMETRY<br/>UNKNOWN                                                                                                              | Not supported yet                                                                                                                                   |
+
+## Sink Options
+
+|               name               |  type   | required |               default value                |                                                                                                                                                                                                                                       Description                                                                                                                                                                                                                                        |
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/HdfsFile.md]: `Name |  Type   | Required  | Xxxx`

File: docs/en/connector-v2/source/HdfsFile.md [hunk 6]
```diff
@@ -1,18 +1,12 @@
...
+## Support Those Engines
 
-If you use spark/flink, In order to use this connector, You must ensure your spark/flink cluster already integrated hadoop. The tested hadoop version is 2.x.
-
-If you use SeaTunnel Engine, It automatically integrated the hadoop jar when you download and install SeaTunnel Engine. You can check the jar package under ${SEATUNNEL_HOME}/lib to confirm this.
-
-:::
+> Spark<br/>
+> Flink<br/>
+> Seatunnel Zeta<br/>
```
Comment (zhilinli123) [inline docs/en/connector-v2/source/HdfsFile.md]: `SeaTunnel Zeta`

File: docs/en/connector-v2/source/HdfsFile.md [hunk 7]
```diff
@@ -1,18 +1,12 @@
...
 
 ## Key features
```
Comment (zhilinli123) [inline docs/en/connector-v2/source/HdfsFile.md]: `Key Features`

File: docs/en/connector-v2/source/HdfsFile.md [hunk 8]
```diff
@@ -33,233 +27,77 @@ Read all the data in a split in a pollNext call. What splits are read will be sa
...
-tyrantlucifer#26#male
+## Supported DataSource Info
+
+| Datasource |  Supported Versions |
+|------------|---------------------|
+| HdfsFile   | hadoop 2.x and 3.x  |
+
+## Data Type Mapping
+
+|                                                          Mysql Data type                                                          |                                                                 Seatunnel Data type                                                                 |
```
Comment (zhilinli123) [inline docs/en/connector-v2/source/HdfsFile.md]: `SeaTunnel Data type`

File: docs/en/connector-v2/source/HdfsFile.md [hunk 9]
```diff
@@ -33,233 +27,77 @@ Read all the data in a split in a pollNext call. What splits are read will be sa
...
+| TINYBLOB<br/>MEDIUMBLOB<br/>BLOB<br/>LONGBLOB<br/>BINARY<br/>VARBINAR<br/>BIT(n)                                                  | BYTES                                                                                                                                               |
+| GEOMETRY<br/>UNKNOWN                                                                                                              | Not supported yet                                                                                                                                   |
+
+## Source Options
+
+|           name            |  type   | required |    default value    |                                                                                                                                                                  Description                                                                                                                                                                  |
```
Comment (zhilinli123) [inline docs/en/connector-v2/source/HdfsFile.md]: ditto

**Comment by lightzhao** [comment]
all update.

**Comment by EricJoy2048** [comment]
Can we add more examples? Some key features, such as kerberos, require specific examples to be added?

**Comment by lightzhao** [comment]
> kerberos

done.

**Comment by lightzhao** [comment]
@EricJoy2048 @zhilinli123 PTAL.

File: docs/en/connector-v2/sink/HdfsFile.md [hunk 10]
```diff
@@ -30,183 +24,108 @@ By default, we use 2PC commit to ensure `exactly-once`
...
 
-Writer the sheet of the workbook
+source {
+  # This is a example source plugin **only for test and demonstrate the feature source plugin**
+  FakeSource {
+    parallelism = 1
+    result_table_name = "fake"
+    row.num = 16
+    schema = {
+      fields {
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/HdfsFile.md:90]: Add all types of tests
Add completed type test thks
@lightzhao

**Comment by ic4y** [comment]
@lightzhao Please merge the latest dev branch and resolve the conflicts.

**Comment by hailin0** [comment]
@lightzhao 
Please fix the code conflict

**Comment by lightzhao** [comment]
> @lightzhao Please fix the code conflict

fixed

File: docs/en/connector-v2/sink/HdfsFile.md [hunk 11]
```diff
@@ -244,32 +161,26 @@ HdfsFile {
...
-  - When field from upstream is null it will throw NullPointerException
-  - Sink columns mapping failed
-  - When restore writer from states getting transaction directly failed
+```
+HdfsFile {
+    fs.defaultFS = "hdfs://hadoopcluster"
+    path = "/tmp/hive/warehouse/test2"
+    kerberos_principal = "your_principal@EXAMPLE.COM"
+    kerberos_keytab_path = "/path/to/your/keytab/file.keytab"
+}
```
Comment (ic4y) [inline docs/en/connector-v2/sink/HdfsFile.md:187]: Add hdfs_site_path , It is usually used in conjunction with hdfs_site_path.

**Comment by ic4y** [comment]
@lightzhao 😄

**Review by EricJoy2048 — DISMISSED** [review]

File: docs/en/connector-v2/sink/HdfsFile.md (hunk 11 above)
Comment (lightzhao) [reply docs/en/connector-v2/sink/HdfsFile.md:187]: thanks. Just prepare to update and see that you have added.

File: docs/en/connector-v2/sink/HdfsFile.md (hunk 10 above)
Comment (lightzhao) [reply docs/en/connector-v2/sink/HdfsFile.md:90]: done.

**Review by ic4y — APPROVED** [review]

**Review by liugddx — APPROVED** [review]
+1

**Pull request merged**